Dockerfile

```
@@ -0,0 +1,64 @@
+
+# Copy source code
+COPY . .
+
+# Build the binary
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build \
+    -ldflags="-w -s -X main.Version=${GIT_VERSION:-dev} -X main.Commit=${GIT_COMMIT:-unknown} -X main.BuildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" \
+    -o sqlc-wizard \
+    cmd/sqlc-wizard/main.go
+
+# Final stage
+FROM alpine:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)